bin/cigar

```diff
@@ -78,17 +80,19 @@ if ( ! file_exists($file)) {
     exit(1);
 }
 
+$secure = ! (isset($options['i']) || isset($options['insecure']));
+$authorization = $options['a'] ?? $options['auth'] ?? null;
+$headers = (array) ($options['h'] ?? $options['header'] ?? []);
+$connectTimeout = $options['connect-timeout'] ?? null;
```

Comment:
This will always be `null` as `connect-timeout` is not in the options anymore (but it's still in the help text)